iwf/tests/worker_server.py

```diff
@@ -0,0 +1,55 @@
+import traceback
+from threading import Thread
+
+from flask import Flask, request
+from iwf_api.models import WorkflowStateWaitUntilRequest, WorkflowStateExecuteRequest
+
+from iwf.registry import Registry
+from iwf.worker_service import (
+    WorkerService,
+)
+
+debug_mode = False
+
+registry = Registry()
+
+_flask_app = Flask(__name__)
+
+_worker_service = WorkerService(registry)
+
+
+@_flask_app.route("/")
+def index():
+    return "hello"
+
+
+@_flask_app.route(WorkerService.api_path_workflow_state_wait_until, methods=["POST"])
+def handle_wait_until():
+    req = WorkflowStateWaitUntilRequest.from_dict(request.json)
+    resp = _worker_service.handle_workflow_state_wait_until(req)
+    return resp.to_dict()
+
+
+@_flask_app.route(WorkerService.api_path_workflow_state_execute, methods=["POST"])
+def handle_execute():
+    req = WorkflowStateExecuteRequest.from_dict(request.json)
+    resp = _worker_service.handle_workflow_state_execute(req)
+    return resp.to_dict()
+
+
+@_flask_app.errorhandler(Exception)
+def internal_error(exception):
+    print("500 error caught")
+    print(traceback.format_exc())
```

Comment:
Consider using the standard logging interface: https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library